### DISCUSSION TITLE
运行时遇到错误ImportError: cannot import name 'BaseDispatcher' from 'graia.broadcast.entities.event'

### DISCUSSION THREAD

**Discussion by OOooo0Oo** [GraiaProject/Application#145] [Q&A]
运行示例时遇到错误

Traceback (most recent call last):
  File "D:\eclipse-workspace\Text-to-Speech\Text-To-Speech-Python\src\TTS\bot.py", line 2, in <module>
    from graia.application import GraiaMiraiApplication, Session
  File "D:\Anaconda3\lib\site-packages\graia\application\__init__.py", line 18, in <module>
    from graia.application.event.lifecycle import (  # for init lifecycle events
  File "D:\Anaconda3\lib\site-packages\graia\application\event\lifecycle.py", line 3, in <module>
    from graia.broadcast.entities.event import Dispatchable, BaseDispatcher
ImportError: cannot import name 'BaseDispatcher' from 'graia.broadcast.entities.event' (D:\Anaconda3\lib\site-packages\graia\broadcast\entities\event.py)

Python是3.8.3版本
麻烦解答一下谢谢

**Comment by karakoo** [comment]
已经写了一个pr，这个bug将在下个版本被修复
若需要的话，可以先看看pr的内容，自行修改源码

**Reply by FridayAnubis** [reply]
目前是否解决？7.27使用pip安装库后依旧出现#145问题，
测试时，库中
D:\\Anaconda\lib\site-packages\graia\application\event\lifecycle.py
无法正常运行
ImportError: cannot import name 'BaseDispatcher' from 'graia.broadcast.entities.event' (D:\Anaconda\lib\site-packages\graia\broadcast\entities\event.py)

**Reply by karakoo** [reply]
有问题的话可以先回退版本
#147
